app/controllers/notifications_controller.rb

```diff
@@ -39,6 +39,10 @@ def index
         changed = current_user.saw_notification_id(max_id)
       end
 
+      if !params.has_key?(:silent) && !@readonly_mode && guardian.can_see_review_queue?
+        current_user.bump_last_seen_reviewable!
+      end
```

Comment:
I have a question here. Why does fetching notifications equate to seeing a reviewable?

Reply:
In the new menu, you get a blue bubble on your avatar if you have unseen reviewables, just like regular notifications in the current menu. The blue bubble for regular notifications goes away as soon as you open the menu (this is done by bumping `seen_notification_id` of the current user), and to make the blue bubble go away when there are new reviewables, we need to bump `last_seen_reviewable_id` when the menu is opened, i.e. when notifications are fetched.

If this doesn't make, let me know and I'll make a screenrecording.

Reply:
O ok. Are we able to defer the bumping of last seen reviewable here? I don't think it should block or add any overhead during the fetching of notifications.

Reply:
yeah this is a good suggestions, I think we can safely wrap it in a `Scheduler::Defer.later` block.

Reply:
done!